src/Card/Card.js

```diff
@@ -39,6 +38,12 @@ class Card extends Component {
             />
             <div className="counter--container">
               <p>{this.props.card.price}</p>
+              <p>
+                {
+                  `You have ${this.props.card.userCount}
+                   of ${this.props.card.requiredCount}`
+                }
+              </p>
```

Comment:
I was trying to break the text up onto two lines so I wouldn't exceed the 80 character limit for a line. But it was removing the space between the word 'of' and the numbers around it. The solution I found was to use a template literal.

Reply:
I like this! It was a little ambiguous before about how they would know what they had as part of that deck.